### PR TITLE
Security docs: Suggest more automatic import of LiveHelpers

### PR DESCRIPTION
**Pull request by henrik**
And some minor copy tweaks.

### CODE REVIEW

**Comment by henrik** [comment]
Side-note:

I don't love how you need to remember to add something to every mount, and if you forget, you're less secure. Might be better with some kind of a "before hook". Let me know if you'd like me to create an issue suggesting the same.

**Comment by josevalim** [comment]
@henrik the reason I am not super worried about having to call it is because, in all of my uses so far, the default helper sets assigns which breaks the LiveView if they are not set anyway. You can also easily solve this by generating this into live views:

```elixir
def mount(..., ..., ...) do
  ...
  |> validate()
  |> verified_mount()
end
```

And your LV implements verified mount instead.

File: lib/phoenix_live_view.ex [hunk 1]
```diff
@@ -806,16 +806,24 @@ defmodule Phoenix.LiveView do
...
-  `MyAppWeb.LiveHelpers`, which you will use and import of every LiveView:
-
-      import MyAppWeb.LiveHelpers
+  `assign_defaults/2` or similar, putting it in a new module like
+  `MyAppWeb.LiveHelpers`, and modifying `lib/my_app_web.ex` so all
+  LiveViews automatically import it:
+  
+      def live_view do
+        quote do
+          # …other stuff…
```
Comment (josevalim) [inline lib/phoenix_live_view.ex:815]:
```suggestion
          # ...other stuff...
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by henrik** [comment]
Thanks, @josevalim! ❤️

I just added very basic “check if a secret key is set in a URL param or session” auth to an internal app, which is only used to restrict access and not to determine a current user. So in this case it would be easy to forget in one LV and nothing would blow up. I’ll try
your suggestion :)

**Pull request merged**